### DISCUSSION TITLE
Make sure when a new commits comes from a starred repository it should have a blue dot next to For you.

### DISCUSSION THREAD

**Discussion by IamNickerson** [community/community#16143] [Other Feature Feedback, Questions, & Ideas]
Hey is it possible that you can add a blue dot next to the for you like you did the with notifications bell? Because it could be great idea incase if any new commits comes out from a repository I starred. Mostly I used the For you to check any commits that came out it will be a great idea to add a blue dot next to it incase if forgot any commit that came out after when I am checking something else.

**Comment by IamNickerson** [comment]
Not only that when a new commit comes it will have blue dot indicating a new commit came.

**Reply by gls1050** [reply]
Good idea
